spirv_msl.hpp

```diff
@@ -152,9 +152,11 @@ class CompilerMSL : public CompilerGLSL
 		Platform platform = macOS;
 		uint32_t msl_version = make_msl_version(1, 2);
 		uint32_t texel_buffer_texture_width = 4096; // Width of 2D Metal textures used as 1D texel buffers
+		uint32_t aux_buffer_index = 0;
```

Comment:
Is there any way to query if this buffer has actually been used after compilation finishes?

Reply:
It should only ever be emitted if it would be used. That's what the `SampledImageScanner` checks for.

Reply:
Right, but the consumer might want to know if it should bind and create the spvAux buffer on the driver side.

Reply:
Guess we could just add this later when we get to integration, it should be trivial to add when needed.